crates/notedeck/src/app.rs

```diff
@@ -102,11 +109,17 @@ fn setup_profiling() {
 }
 
 impl Notedeck {
-    pub fn new<P: AsRef<Path>>(ctx: &egui::Context, data_path: P, args: &[String]) -> Self {
+    pub fn new<P: AsRef<Path>>(
+        ctx: &egui::Context,
+        data_path: P,
+        args: &[String],
+    ) -> (Self, BTreeSet<String>) {
```

Comment:
I don't want to complicate this interface since other apps may use it. let's just store the set on the struct and access it with a method.